qa/tasks/cephfs/test_scrub_checks.py

```diff
@@ -443,6 +444,57 @@ def test_scrub_split_dirfrags(self):
             timeout=30
         )
 
+    def test_scrub_backtrace_divergent_newer(self):
+        """
+        The consequence of the scrub will not acquire nest locks which would update
+        the iversion for the in-memory inode prior to comparing to on-disk values.
+        """
+
+        self.fs.set_max_mds(2)
+        self.fs.wait_for_daemons()
+
+        test_path1 = "testdir1"
+        test_path2 = "testdir1/testdir2"
+        test_path3 = f"{test_path2}/testdir3"
+        test_path4 = f"{test_path3}/testdir4"
+        test_file = f"{test_path3}/file_0"
+
+        # Make sure all the ancestors' auth is in rank 0
+        self.mount_a.run_shell(["mkdir", test_path1])
+        self.mount_a.setfattr(test_path1, "ceph.dir.pin", "0") # no surprise migrations
+        self.mount_a.run_shell(["mkdir", "-p", test_path4])
+
+        # Will trigger to load ancestor inodes in mds.1
+        self.mount_a.setfattr(test_path4, "ceph.dir.pin", "1")
+        # Will trigger to load ancestor inodes in mds.1
```

Comment:
I don't understand this comment. `test_path4` will eventually exported because it's pinned but not right now because it's empty.